docs/clients_openai.md

```diff
@@ -67,8 +67,45 @@ curl http://localhost:8000/v3/chat/completions \
 :::
 ::::
 
+### Request completions with unary calls
 
-### Request chat completion with streaming
+::::{tab-set}
+:::{tab-item} python [OpenAI] 
+:sync: python-openai
+```{code} python
+from openai import OpenAI
+client = OpenAI(base_url="http://localhost:8000/v3", api_key="unused")
+response = client.completions.create(
+    model="meta-llama/Llama-2-7b",
```

Comment:
shall we use llama2 and not 3 as in the demo?

Reply:
fixed